# Dylan C. Gagler
Computational Biologist | Astrobiologist

[CV](https://github.com/dgagler/dgagler/blob/master/dgagler_CV.pdf)

[LinkedIn](https://www.linkedin.com/in/dylan-gagler-4a0a68191/) 

## About Me
I'm a master's student at Arizona State University, studying astrobiology and computational biology in the lab of Dr. Sara Imari Walker. I am fluent in Python and have particular expertise analyzing and visualizating complex biological data. My thesis involves accumulating large amounts of genetic and biochemical data from microorganisms to uncover large scale patterns in enzyme biochemistry, and a manuscript is currently in prep. Prior research experience includes dissolved gas chromatography, DNA extraction and amplification, 16S RNA analysis, and microbial taxonomic characterization using the R package, QIIME 2. Pre-prior research experience includes geoarchaeological and paleontological field work in Kenya, and geochronological analysis of detrital zircons using Laser Ablation Inductively Coupled Plasma Mass Spectrometry (LA-ICP-MS). Outside of research, I love to play table tennis, explore nature, read, and spend time with the people I love.

# Data Visualization Portfolio
Here you can see some of the plots and figures I've generated.

# Projects
Below you can learn more about my research and previous and current projects. Click on the title links to see overviews in Jupyter NBViewer.

# [Patterns in life's use of major enzymatic reaction classes](https://nbviewer.jupyter.org/github/dgagler/dgagler.github.io/blob/master/enzyme_demo.ipynb)

**Data retrieval, cleaning, analysis, and visualization in python.** A driving question in astrobiology is whether the structure and diversity of life we see on Earth is the result of chemical and evolutionary contingency, or the manifestation of underlying organizational principles. The realization of such principles would inform attempts at synthesizing life in the laboratory, searching for life in the universe, and understanding the origin of life on Earth. For this project, I leverage large amounts of publically available genetic and enzymatic data for organisms across the tree of life in tandem with biochemical reaction data to uncover patterns in life's use of enzyme reaction classes across domains and levels of organization.

# [Microbial Exploration of Spirea Creek, Yellowstone National Park](https://nbviewer.jupyter.org/github/dgagler/spirea/blob/master/spirea_sequencing_demo.ipynb)

**Microbiological fieldwork, DNA extraction and PCR amplification, sequence analysis and taxonomic classification using QIIME 2, and data visualization in python.** Yellowstone National Park (YNP) is one of the premier locations to study the intersection of geochemistry and microbiology. For this project, I collected microbial samples from a set of previously uncharacterized geochemical hot springs in YNP. DNA was extracted from these samples, amplified via PCR, and subsequently sequenced. I analyzed the sequence data using QIIME 2 and visualized the results using python. This project explores the biology of diverse hot spring ecosystems.
